src/Model/ModelManager.php

```diff
@@ -277,6 +277,13 @@ public function executeQuery($query)
             __METHOD__
         ), E_USER_DEPRECATED);
 
+        //throw new \InvalidArgumentException(sprintf(
```

Comment:
This should be `\TypeError`.

Reply:
👍 fixed in https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/481